monai/metrics/meandice.py

```diff
@@ -104,6 +113,9 @@ def compute_meandice(y_pred: torch.Tensor, y: torch.Tensor, include_background:
             The values should be binarized.
         include_background: whether to skip Dice computation on the first channel of
             the predicted output. Defaults to True.
+        ignore_empty: whether to ignore empty ground truth cases during calculation.
+            If `True`, NaN value will be set for empty ground truth cases.
+            If `False`, 1 will be set if the predictions of empty ground truth cases are also empty.
```

Comment:
Hi @yiheng-wang-nv, I think we can provide a more generic option of `empty_default`: the Dice score to use when the ground truth class is missing, default to `"nan"`.

then in the code:

```py
default_val = torch.tensor(float(empty_default), device=y_o.device)
return torch.where(denominator > 0, (2.0 * intersection) / denominator, default_val)
```

Reply:
Sorry I misunderstood the where condition perhaps we don't need this default value option for now.

Reply:
Sorry I misunderstood the where condition perhaps we don't need this default value option for now.